### PR TITLE
Changed F.activation func to torch.activation func

### PR DESCRIPTION
**Pull request by developer0hye**
Changed F.activation func to torch.activation func

User Warning is detected when I run train.py.

UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.

### CODE REVIEW

**Comment by developer0hye** [comment]
@ShuYingShu 
sorry, it was my mistake!

**Comment by developer0hye** [comment]
@ShuYingShu 
Thank you!
I modified the code and do [pr](https://github.com/leftthomas/SRGAN/pull/50)!
